setup.py

```diff
@@ -279,10 +279,10 @@ def run(self):
     'Intended Audience :: Developers',
     'Topic :: Software Development :: Testing',
     'License :: OSI Approved :: BSD License',
-    'Programming Language :: Python :: 3.7',
     'Programming Language :: Python :: 3.8',
     'Programming Language :: Python :: 3.9',
     'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
```

Comment:
`python_requires` item further down needs bumping

Reply:
Done in https://github.com/adobe-type-tools/pyots/pull/25/commits/b43a091a21bea491603721609a91fea9882688d4